diff --git a/CMakeLists.txt b/CMakeLists.txt
index 50f5e3799..218d77f55 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -686,13 +686,13 @@ add_executable(crypto_test ${CRYPTO_TEST_SOURCES})
 target_link_libraries(crypto_test test_support_lib boringssl_gtest crypto)
 add_dependencies(all_tests crypto_test)
 
-add_executable(ssl_test ${SSL_TEST_SOURCES})
-target_link_libraries(ssl_test test_support_lib boringssl_gtest ssl crypto)
-add_dependencies(all_tests ssl_test)
-add_executable(decrepit_test ${DECREPIT_TEST_SOURCES})
-target_link_libraries(decrepit_test test_support_lib boringssl_gtest
-                      decrepit crypto)
-add_dependencies(all_tests decrepit_test)
+# add_executable(ssl_test ${SSL_TEST_SOURCES})
+# target_link_libraries(ssl_test test_support_lib boringssl_gtest ssl crypto)
+# add_dependencies(all_tests ssl_test)
+# add_executable(decrepit_test ${DECREPIT_TEST_SOURCES})
+# target_link_libraries(decrepit_test test_support_lib boringssl_gtest
+#                       decrepit crypto)
+# add_dependencies(all_tests decrepit_test)
 
 if(APPLE)
   set(PKI_CXX_FLAGS "-fno-aligned-new")
@@ -717,8 +717,8 @@ target_link_libraries(bssl ssl crypto)
 # existing tools, we, for now, copy the targets into the subdirectories. This
 # will be removed sometime in 2024.
 copy_post_build(crypto crypto crypto_test urandom_test)
-copy_post_build(ssl ssl ssl_test)
-copy_post_build(decrepit decrepit decrepit_test)
+# copy_post_build(ssl ssl ssl_test)
+# copy_post_build(decrepit decrepit decrepit_test)
 copy_post_build(tool bssl)
 
 if(FUZZ)
